buckets/redis/bucket_factory.go

```diff
@@ -50,19 +50,24 @@ type bucketFactory struct {
 	redisOpts         *redis.Options
 	scriptSHA         string
 	connectionRetries int
+	flushdbCommand    string
 }
 
 // NewBucketFactory creates a new bucketFactory instance
-func NewBucketFactory(redisOpts *redis.Options, connectionRetries int) quotaservice.BucketFactory {
+func NewBucketFactory(redisOpts *redis.Options, connectionRetries int, flushdbCommand string) quotaservice.BucketFactory {
```

Comment:
Maybe some docs here that suggest that if `flushdbCommand` is `""` then `flushdb`  is the default?